rust/xaynet-server/src/state_machine/coordinator.rs

```diff
@@ -64,7 +65,7 @@ impl CoordinatorState {
             max_sum_time: pet_settings.max_sum_time,
             max_update_time: pet_settings.max_update_time,
             mask_config,
-            model_size: model_settings.size,
+            model_length: model_settings.length,
```

Comment:
I wonder if we can actually remove `mask_config` and `model_length` because they are now part of the round parameters.